src/routes/safe/components/Transactions/TxList/Filter/utils.ts

```diff
@@ -34,3 +52,66 @@ export const getFilterHelperText = (value: string, error?: FieldError): TextFiel
 
   return undefined
 }
+
+// Filter helper
+
+export const isTxFilter = (object: ParsedUrlQuery): object is Partial<FilterForm> => {
```

Comment:
Did you test what happens if I manually enter one of this query params ? Or if I enter it with invalid values? For the case the url is not generated from the Filter form

Reply:
After our discussion, I've left this as `some`, there only needs to be part of the object for a valid filter to exist. The endpoint-specific filter is created in `getHistoryTxListPage` from the object that this function checks.

I've also removed the hidden fields from the URL.

Reply:
After checking it together it is no harm to send invalid keys or values to the backend. They will simply be ignored or not applied.